cobigen/cobigen-core-parent/cobigen-core/src/main/java/com/devonfw/cobigen/impl/util/TemplatesClassloaderUtil.java

```diff
@@ -298,9 +298,9 @@ public FileVisitResult visitFile(Path file, BasicFileAttributes attrs) throws IO
             public FileVisitResult visitFileFailed(Path file, IOException exc) throws IOException {
                 // Log errors but do not throw an exception
                 LOG.warn("An IOException occurred while reading file on path {} with message: {}", file,
-                    exc.getMessage());
+                    exc.getMessage(), LOG.isDebugEnabled() ? exc : null);
                 LOG.debug("An IOException occurred while reading file on path {} with message: {}", file,
-                    LOG.isDebugEnabled() ? exc : null);
+                    exc.getMessage(), LOG.isDebugEnabled() ? exc : null);
```

Comment:
remove duplicated debug log as this is already covered by warn

Reply:
adjusted.